src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -164,7 +164,7 @@ public PersistentDataStore() {
     List<Event> events = new ArrayList<>();
 
     // Retrieve all events from the datastore.
-    Query query = new Query("chat-events").addSort("creation_time", SortDirection.DESCENDING);
+    Query query = new Query("chat-events").addSort("creation_time", SortDirection.ASCENDING);
```

Comment:
I believe that the events will always be in order, but I based this off of the original code. Should I remove it?

Reply:
I think we should only keep it if we know that it adds something. Otherwise, let's at least remove it and see if it breaks our tests, which should make sure events are sorted.

Reply:
The test that saves and loads events fails after removing this.